### ISSUE TITLE
PvP Leaderboards missing

### ISSUE THREAD

**Issue by oliv-tree** [benweier/blizzard.js#177]

**Comment by benweier** [comment]
Hey @oliv-tree, the PVP Season API is currently available on the retail client but not on the classic client. Checking the documentation it does look like it is supported by the classic game data APIs now so I may need to add this to the lib

**Comment by oliv-tree** [comment]
> Hey @oliv-tree, the PVP Season API is currently available on the retail client but not on the classic client. Checking the documentation it does look like it is supported by the classic game data APIs now so I may need to add this to the lib

Again this was my bad, however I think the implementation is quite confusing. I.e., how you would reach the code for each of these is not very clear from the docs/types.
```ts
let leaderboardIndex = await wowClient.pvpSeason({
  id: 36,
  resource: "leaderboard",
})
```
```ts
let leaderboard = await wowClient.pvpSeason({
  id: 36,
  bracket: "shuffle-demonhunter-havoc",
  resource: "leaderboard",
})
```
```ts
let season = await wowClient.pvpSeason({
  id: 36,
})
```
```ts
let seasonRewards = await wowClient.pvpSeason({
  id: 36,
  resource: "reward",
})
```

**Comment by benweier** [comment]
I agree it's not as clear as it could be and there's an opportunity here to improve the documentation
